tools/marvin/marvin/config/test_data.py

```diff
@@ -1847,119 +1847,187 @@
             },
-            "upload_volume": {
-                "diskname": "UploadVol",
-                "format": "VHD",
-                "url": "http://download.cloudstack.org/releases/2.0.0/UbuntuServer-10-04-64bit.vhd.bz2",
-                "checksum": "",
+            "xenserver": {
+                "url": ""
             },
-            "bootableIso":
-                {
-                    "displaytext": "Test Bootable ISO",
-                    "name": "testISO",
-                    "bootable": True,
-                    "ispublic": False,
-                    "url": "http://dl.openvm.eu/cloudstack/iso/TinyCore-8.0.iso",
-                    "ostype": 'Other Linux (64-bit)',
-                    "mode": 'HTTP_DOWNLOAD'
-                },
-            "setHostConfigurationForIngressRule": False,
-            "restartManagementServerThroughTestCase": False,
-            "vmxnet3template": {
-                "displaytext": "VMXNET3 Template",
-                "name": "VMXNET3 template",
-                "ostype": "CentOS 5.6 (64-bit)",
-                "isfeatured": True,
-                "ispublic": False,
-                "isextractable": True,
-                "mode": "HTTP_DOWNLOAD",
-                "templatefilter": "self",
-                "url": "http://people.apache.org/~sanjeev/systemvm64template-2014-09-30-4.3-vmware.ova",
-                "hypervisor": "vmware",
-                "format": "OVA",
-                "nicadapter": "vmxnet3",
-                "kvm": {
-                    "url": ""
-                },
-                "vmware": {
-                    "url": ""
-                },
-                "xenserver": {
-                    "url": ""
-                },
-                "hyperv": {
-                    "url": ""
-                },
-                "ostype": 'CentOS 5.3 (64-bit)',
-                "mode": 'HTTP_DOWNLOAD'
-            }
+            "hyperv": {
+                "url": ""
+            },
+            "ostype": 'CentOS 5.3 (64-bit)',
+            "mode": 'HTTP_DOWNLOAD'
+        }
+    },
+    "cks_kubernetes_versions": {
+        "1.14.9": {
+            "semanticversion": "1.14.9",
+            "url": "http://download.cloudstack.org/cks/setup-1.14.9.iso",
+            "mincpunumber": 2,
+            "minmemory": 2048
+        },
+        "1.15.0": {
+            "semanticversion": "1.15.0",
+            "url": "http://download.cloudstack.org/cks/setup-1.15.0.iso",
+            "mincpunumber": 2,
+            "minmemory": 2048
+        },
+        "1.16.0": {
+            "semanticversion": "1.16.0",
+            "url": "http://download.cloudstack.org/cks/setup-1.16.0.iso",
+            "mincpunumber": 2,
+            "minmemory": 2048
+        },
+        "1.16.3": {
+            "semanticversion": "1.16.3",
+            "url": "http://download.cloudstack.org/cks/setup-1.16.3.iso",
+            "mincpunumber": 2,
+            "minmemory": 2048
         }
+    },
+    "cks_templates": {
```

Comment:
@shwstppr you'll need to sync changes in test_data in trillian specific test_data as well

Reply:
Have done that